packages/iTwinUI-react/src/core/Dialog/DialogMain.tsx

```diff
@@ -210,6 +209,7 @@ export const DialogMain = React.forwardRef<HTMLDivElement, DialogMainProps>(
           <Resizer
             elementRef={dialogRef}
             containerRef={dialogContext.dialogRootRef}
+            onResizeStart={() => setResizeStyle({ maxWidth: '100%' })}
```

Comment:
This only needs to happen once so i would add guard this in an if condition (can use a boolean ref)

Reply:
Thanks, added a guard condition using a boolean ref

Reply:
Please remove the console log

Reply:
[DONE] Oops! :smile: Removed